website/pages/docs/commands/operator/raft.mdx

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs
+page_title: operator raft - Command
+sidebar_title: <code>raft</code>
+description: >-
+  The "operator raft" command is used to interact with the integrated Raft storage backend.
+---
+
+# operator raft
+
+This command groups subcommands for operators to manage the integrated raft storage backend.
+
+```text
+Usage: vault operator raft <subcommand> [options] [args]
+
+ This command groups subcommands for operators interacting with the Vault raft
+ storage backend. Most users will not need to interact with these commands. Here
+ are a few examples of the raft operator commands:
+
+Subcommands:
+    join           Joins a node to the raft cluster
+    list-peers     Returns the raft peer set
+    remove-peer    Removes a node from the raft cluster
+    snapshot       Restores and saves snapshots from the raft cluster
+```
+
+## join
+
+This command is used to join a new node as a peer to the raft cluster.
```

Comment:
```suggestion
This command is used to join a new node as a peer to the Raft cluster. In order to join, there must be at least one existing member of the cluster. Agents can join other agents multiple times without issue. If a node that is already part of a cluster joins another node, then the clusters of the two nodes join to become a single cluster.
```
I think it would be nice to include more information generally.

Reply:
Some of the things in ☝️ are applicable only to Consul. Took the first sentence though. Thanks!